libdivide.h

```diff
@@ -344,10 +344,8 @@ static LIBDIVIDE_INLINE int64_t libdivide_mullhi_s64(int64_t x, int64_t y) {
 
 static LIBDIVIDE_INLINE int32_t libdivide_count_leading_zeros32(uint32_t val) {
 #if defined(__AVR__)
-    // Fast way to count leading zeros
-    // On the AVR 8-bit architecture __builtin_clz() works on a int16_t.
-    // So use __builtin_clzll() & correct back to the 32-bit equivalent.
-    return __builtin_clzll(val)-32;
+   // Fast way to count leading zeros
+    return __builtin_clzl(val);
```

Comment:
Please squash this commit into its parent.

Reply:
Done - all squashed into a single commit